kibana-reports/public/components/report_definitions/report_settings/__tests__/report_settings_helpers.test.tsx

```diff
@@ -51,18 +51,26 @@ describe('report_settings_helpers tests', () => {
   });
 
   test('getVisualizationBaseUrlCreate', () => {
-    const baseUrl = getVisualizationBaseUrlCreate(true, true);
+    const baseUrl = getVisualizationBaseUrlCreate(true, '12345', true);
```

Comment:
what is this hardcode value for

Reply:
sorry, it's in test files, I see

Reply:
This is just a hard-code value for testing to fulfill the parameter- since this is a unit test we don't have an actual report definition id to use

Reply:
can you make it a constant tho, since you are using it at more than one place

Reply:
Sure, will use a constant